frontend/src/core/wasm/worker/bootstrap.ts

```diff
@@ -206,6 +206,10 @@ export class DefaultWasmController implements WasmController {
 
   private async loadNotebookDeps(code: string, foundImports: Set<string>) {
     const pyodide = this.requirePyodide;
+    // Additional dependencies of marimo that are lazily loaded.
+    //
+    // pyodide-http is a patch to make basic http requests work in pyodide
+    await pyodide.loadPackage(["docutils", "Pygments", "jedi", "pyodide-http"]);
```

Comment:
Does this actually make it faster? This is now serial instead of parallel

Reply:
Oh I see it shows the notebook code faster but not started. You can still push this down to be parallel with the users deps

Reply:
Yea, it makes the code show faster. Will do, thanks for the tip

Reply:
You can promise.all this below at line 224